libclamav/yara_arena.c

```diff
@@ -591,7 +591,7 @@ int yr_arena_allocate_memory(
 // yr_arena_allocate_struct
 //
 // Allocates a structure within the arena. This function is similar to
-// yr_arena_allocate_memory but additionaly receives a variable-length
+// yr_arena_allocate_memory but additionally receives a variable-length
```

Comment:
This is 3rd party code, so as tempting as it is to fix, we should leave it as-is.
It looks like they've fix it since then in their repo: https://github.com/VirusTotal/yara/blob/master/libyara/arena.c#L343
We plan to update our copy, but haven't found the time in recent years ☹

Reply:
Thanks restored the original